TelCo.ColorCoder/AnyExceptions.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using TelCo.ColorCoderClasses;
+
+namespace TelCo.ColorCoderExecptions
+{
+    /// <summary>
+    /// This class contains different exception conditions and throw respective exceptions.
+    /// </summary>
+    public class AnyExceptions
```

Comment:
what do you mean by Name - AnyExceptions ?  
Looks like class AnyExceptions interested in Validating data

Reply:
Updated Changes: Renamed the file